dfstools/src/relationship_tools.py

```diff
@@ -1,6 +1,37 @@
 import pandas as pd
 
 
+def pecan_cookies_load_data():
+    pd.set_option("display.max_rows", None, "display.max_columns", None)
+    pd.set_option('display.width', 15000)
+
+    airports = pd.read_csv("./data/airports/airports.csv")
```

Comment:
This will eventually get you into trouble based on your current working directory when you execute your code, especially during unit testing since that can be executed from several locations. Take a look at [this](https://github.com/NCATComp410/comp410_summer2020/blob/9a87027ade043fca5043c75f8d7311c51341f4ce/dfstools/tests/test_relationship_tools.py#L39) code for some possible improvements.

Reply:
Okay thank you! I guess I need to make sure I use the `os.path.join()` function to differentiate between operating systems